### PR TITLE
Change token to APIKey in Configuration service API

### PR DESCRIPTION
**Pull request by stingion**

### CODE REVIEW

**Review by ronenhamias — DISMISSED** [review]
Object apiKey should not be String apiKey?

**Comment by stingion** [comment]
Yes, apiKey is String under hook. But from the beginning it was defined as Object for fast substitution if needed.

**Comment by dmytro-lazebnyi** [comment]
@stingion let's change `apiKey` type from `Object` to `String` and remove redundant call `toString`.

File: ApiDocs/create.apidoc [hunk 1]
```diff
@@ -21,7 +21,7 @@
   *         "q":"/configuration/createRepository",
   *         "sid": 1,
   *         "d":{
-  *         "token": "API-TOKEN",
+  *         "apiKey": "API-KEY",
```
Comment (dmytro-lazebnyi) [inline ApiDocs/create.apidoc]: I assume in `apidoc` it should be `APIKey` instead of `apiKey`, since you are using `@JsonProperty("APIKey")`

Comment (stingion) [reply ApiDocs/create.apidoc]: Ok, I'll change

**Review by dmytro-lazebnyi — DISMISSED** [review]

**Review by PavloEsco — APPROVED** [review]
Tested OK on `develop`

**Pull request merged**